ballista/rust/executor/src/main.rs

```diff
@@ -112,6 +116,21 @@ async fn main() -> Result<()> {
         .context("Could not connect to scheduler")?;
 
     let scheduler_policy = opt.task_scheduling_policy;
+    let cleanup_ttl = opt.executor_cleanup_ttl;
+
+    if opt.executor_cleanup_enable {
+        let interval = opt.executor_cleanup_interval;
+        let mut interval_time = time::interval(Core_Duration::from_secs(interval));
+        tokio::spawn(async move {
+            loop {
+                interval_time.tick().await;
+                clean_shuffle_data_loop(&work_dir, cleanup_ttl)
+                    .await
+                    .unwrap();
```

Comment:
I am thinking if we should handle error here, or the cleanup thread may be killed if panic.

Reply:
Thanks!